### PR TITLE
[ES] content/es/_TEMPLATE.md

### LINKED ISSUE #1778: [Spanish] localize 'Definition Template'
Localize the `content/en/_TEMPLATE.md` document to Spanish

### PR DESCRIPTION
**Pull request by marianogg9**
### Describe your changes
- added `es/_TEMPLATE.md`.

### Related issue number or link (ex: `resolves #issue-number`)
- resolves #1778 

### Checklist before opening this PR (put `x` in the checkboxes)
- [X] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [X] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/es/_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+title: Plantilla de Definición
+status: Feedback Appreciated
+category: concepto
```
Comment (electrocucaracha) [inline content/es/_TEMPLATE.md:4]:
```suggestion
category: Concepto
```

File: content/es/_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+---
+
+## ¿Qué es?
+
+Resumen rápido del concepto y qué es.
+
+## Problema que aborda
+
+Definir el problema que aborda. Idealmente, no mencionar el término que se está definiendo.
```
Comment (electrocucaracha) [inline content/es/_TEMPLATE.md:13]:
```suggestion
Definir el problema que aborda. Idealmente, no se debe mencionar el término que se está definiendo.
```

**Review by electrocucaracha — CHANGES_REQUESTED** [review]

**Review by ramrodo — APPROVED** [review]
lgtm

**Review by electrocucaracha — APPROVED** [review]
/lgtm

**Pull request merged**